config/routes.rb

```diff
@@ -20,6 +20,8 @@ def matches?(request)
 
   resources :coronavirus_pages, path: "coronavirus", only: %i[index show update], param: :slug do
     get "prepare", on: :member
+    get "reorder", on: :member
+    post "reorder", on: :member
```

Comment:
As stated previously - `post` and `get` should not point at the same action as that leads to the action having to be far more complicated than it would be if it only had to handle a single action.